src/app/Service/ProjectServices.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios';
+import Cookies from 'js-cookie';
+import api from './api';
+
+
+  interface ProjectData {
+    title: string;
+    targetLanguages: string[];
+  }
+
+  export const createProject = async (projectData:ProjectData)=> {
+    const token = Cookies.get("authToken");
+    console.log(projectData)
+    const response = await api.post('/projects',projectData,{
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    console.log("createProject",response.data)
+      return response.data;
+
+  }
+
+  export const getAllProjects = async () =>{
+    const token = Cookies.get("authToken");
+    const response = await api.get("/projects",{
+        headers: {
+            Authorization: `Bearer ${token}`,
+          },
+    })
+    console.log("getAllProjects",response.data)
+    return response.data
+}
+
+
+export const getProjectsById = async (id:string) =>{
+  const token = Cookies.get("authToken");
```

Comment:
if you get it in the top of the file and just pass it in the header of all you won't need to repeat this line in each function.

but the more better is that you can create a middleware that add token to the header of any request without you write which is a great thing next.js bring to us
please learn about it is more useful.